api/handlers/container/restart.go

```diff
@@ -22,8 +24,22 @@ func (h *handler) restart(w http.ResponseWriter, r *http.Request) {
 	}
 	timeout := time.Second * time.Duration(t)
 
+	devNull, err := os.OpenFile("/dev/null", os.O_WRONLY, 0600)
+	if err != nil {
+		response.JSON(w, http.StatusBadRequest, response.NewErrorFromMsg("failed to open /dev/null"))
+		return
+	}
+	defer devNull.Close()
+
 	ctx := namespaces.WithNamespace(r.Context(), h.Config.Namespace)
-	err = h.service.Restart(ctx, cid, timeout)
+	containers := [1]string{cid}
```

Comment:
May just pass cid directly, if we are not planning to support multiple containers